src/js/graph/parsers/graphParserXml.js

```diff
@@ -86,6 +86,7 @@ GraphParserXml.prototype.parse = function(readableStream) {
                     }
                     break;
                 case 'intFeat':
+                case 'enumProp':
```

Comment:
If it's acceptable, I'll add that test to PAGI-162 since the behavior changes immediately to behave like a strProp